NDTensors/src/Unwrap/src/functions/permutedims.jl

```diff
@@ -0,0 +1,3 @@
+function permutedims(E::Exposed{<:AbstractArray}, perm)
```

Comment:
For simplicity we could just use `::Exposed` instead of `::Exposed{<:AbstractArray}`.